### PR TITLE
feat: introduce "release cycle" page and align beta banners

### PR DESCRIPTION
**Pull request by loujaybee**
**Motivation** 

1. Help set user expectations around feature status, API stability etc. 
1. We have inconsistent and mixed language use around new features in the docs
1. To link release cycle status with the changelog ([context](https://github.com/gitpod-io/website/pull/2098#discussion_r880756412))
1. Align the product release cycle callouts on docs pages

**Changes** 

1. Introduce a "releases page" to define alpha/beta/ga in Gitpod context
1. Update all beta callouts to link back to a release pages for context
1. Update all pages to link to a GitHub issue for feedback (some were linking to support)
1. Update all pages to format the beta banner in the same way 
1. Update all pages to ensure beta banner is positioned the same (below title)

Closes https://github.com/gitpod-io/website/issues/2054

|Before|After|
|-|-|
|image|image||image|image|
|image|image|
|image|image|
|image|image|

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2124"></a>

---

- [ ] Decide on what to do with: https://3000-gitpodio-website-45oulknpi21.ws-eu45.gitpod.io/docs/teams
- [ ] Document "soon"
- [ ] Document ["experimental" features](https://www.gitpod.io/docs/ides-and-editors/goland#preconfigure-for-repository)

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2124"></a>

### CODE REVIEW

File: generators/ide/jb-product.md [hunk 1]
```diff
@@ -15,9 +15,7 @@ title: __productTitle__
 <!-- prettier-ignore -->
 # __productFullyQualifiedTitle__
 
-Gitpod enables code editing using JetBrains {title} currently via [JetBrains Gateway](jetbrains-gateway).
```
Comment (loujaybee) [inline generators/ide/jb-product.md:18]: Decided just to remove this line, doesn't seem needed with getting started info below.

File: gitpod/docs/teams-and-projects.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
 ---
 section: develop
-title: Create a Team
+title: Teams & Projects
```
Comment (loujaybee) [inline gitpod/docs/teams-and-projects.md:3]: Updated mismatched title, seems to only affect the tab title.

**Comment by loujaybee** [comment]
@akosyakov - One thing I'd like to clean up here is also the note of "experimental" features. Should we mark these as alpha/beta, or document "experimental" with separate expectations? WDYT? 

e.g. prebuilds and plugin support for JetBrains. 

image

File: gitpod/docs/references/gitpod-releases.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Information regarding how, and when Gitpod releases updates.
+
+For information on latest changes, please see the [changelog](/changelog).
+
+## Gitpod Release Cycle
+
+1. **Alpha** - Feature and published APIs unstable.
+2. **Beta** - Feature and published APIs are relatively stable.
+3. **General Availability (GA)** - Features and published API's are stable. Deprecations and API changes will be communicated in advance.
```
Comment (loujaybee) [inline gitpod/docs/references/gitpod-releases.md:20]: @lucasvaltl - I haven't added it here, but we may want to use this page as somewhere to explain the self-hosted release cycle and how it aligns/differs to gitpod.io.

File: gitpod/docs/references/gitpod-releases.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  export const prerender = true;
+</script>
+
+# Gitpod Releases
+
+Information regarding how, and when Gitpod releases updates.
+
+For information on latest changes, please see the [changelog](/changelog).
+
+## Gitpod Release Cycle
```
Comment (loujaybee) [inline gitpod/docs/references/gitpod-releases.md:16]: **To consider:** 

1. "Experimental" - This is used in a few places, unsure of the difference between this terminology, and alpha/beta.
1. "Soon" - Being used for upcoming features in JetBrains. 
1. "Deprecated"

**Comment by akosyakov** [comment]
> @akosyakov - One thing I'd like to clean up here is also the note of "experimental" features. Should we mark these as alpha/beta, or document "experimental" with separate expectations? WDYT?

Prebuilds are sure alpha, we need to rewrite them to work properly. Right now they don't respect any our configs, like plugins, versions and so on. Plugins support I would say is Beta.

**Review by gtsiolis — APPROVED** [review]
Thanks for making this change, @loujaybee! ✨ 

This is also something we'll tackle in the dashboard soon, as you mentioned in https://github.com/gitpod-io/website/issues/2054#issuecomment-1127462928, and the structure you've added here is close to what I had in mind for the dashboard.

**Pull request merged**